File: FoodApp/app/src/main/java/com/example/foodapp/model/order/Product.kt
package com.example.foodapp.model.order

data class Product(
    val calo: Int,
    val category: List<Category>,
    val created_time: String,
    val description: String,
    val discount: Int,
    val id: Int,
    val images: List<Image>,
    val ingredient: String,
    val name: String,
    val price: Int,
    val quantity: Int,
    val updated_time: String
)

File: admin/app/Models/Product.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Product extends Model
{
    use HasFactory;
    protected $table = "product";
    public $timestamps = false;

    protected $fillable = [
        'name', // Thêm trường 'name' vào mảng fillable
        'description',
        'ingredient',
        'calo',
        'quantity',
        'price',
        'discount',
    ];

    public function category()
    {
        return $this->belongsToMany(Category::class, 'product_category', 'product_id', 'category_id');
    }

    public function images()
    {
        return $this->hasMany(Image::class);
    }

}
